client/simple/src/js/main/preferences.js

```diff
@@ -1,52 +0,0 @@
-/* SPDX-License-Identifier: AGPL-3.0-or-later */
-((_w, d, searxng) => {
-  if (searxng.endpoint !== "preferences") {
-    return;
-  }
-
-  searxng.ready(() => {
-    let engine_descriptions = null;
-
-    function load_engine_descriptions() {
-      if (engine_descriptions == null) {
```

Comment:
Related to https://github.com/searxng/searxng/pull/5073/files#r2256037965

Here (in this closure) the json from `http://127.0.0.1:8888/engine_descriptions.json` was stored in the variable `engine_descriptions` to avoid a reload on every `mouseenter` event.